src/Clap-Core.package/ClapMatcher.class/README.md

```diff
@@ -0,0 +1 @@
+I am an abstract class which stocks many ClapCommand
```

Comment:
Stock is a noun not a verb; the correct word would be *store*.
However the goal of `ClapMatcher` is not to store commands; it's not a persistence class, but more precisely the entry point to the parsing algorithm